### DISCUSSION TITLE
Clockwork Beta Devlog - May 15, 2023

### DISCUSSION THREAD

**Discussion by ToasterPanic** [red-stone-network/clockwork#49] [Announcements]
Hello everyone! School is very close to being out for me right now! As of right now I have a lot of work to do, so don't expect much anytime soon, but Clockwork v2 should be out within the next week or two.

First of all, we've got alternative methods for opening Clockwork now! Right now we have 3 methods - about:blanker, files, and data:// urls. More may come if requested.

Second of all, we now have the plugin system, which allows developers to run JavaScript directly in Clockwork! There's no way to install them yet (without dev console / bookmarklets), but there will be soon.

Finally, we updated our home page to use an SVG as the banner, rather than a WEBP, because the encoding looked weird.

Thanks for reading!

**Comment by CJBrandi** [comment]
when is this coming out? its been 2 weeks already
